xxh3.h

```diff
@@ -1271,6 +1328,14 @@ XXH3_scrambleAcc(void* XXH_RESTRICT acc, const void* XXH_RESTRICT secret)
 
 #define XXH_PREFETCH_DIST 384
 
+#ifdef __clang__ // for clang
```

Comment:
As a quick comment,
I'm surprised to notice that the optimal distance seems to depend on the compiler.
I guess it implies that generated assembly is different enough to warrant such a distinction.

Reply:
yes, i think each compiler will have very different loop cycles, the slower loop will correspond the larger distance, so clang < gcc 64b < gcc 128b.